build.gradle

```diff
@@ -363,6 +380,12 @@ repositories {
             includeGroup 'appeng'
         }
     }
+    maven {
+        url 'https://maven.blamejared.com'
+        content {
+            includeGroup 'vazkii.patchouli'
+        }
+    }
```

Comment:
The includeGroup should just be moved into the existing one for jared's maven with CrT and JEITweaker